### PR TITLE
LTP: Fix for ptrace03

### PR DESCRIPTION
**Pull request by ashokchandran**
Issue: Test was failing as it requires child process for ptrace calls. 
Solution: Removed the wait that was causing test to hang and also disabled one test for PTRACE_TRACEME that requires child process.

### CODE REVIEW

File: tests/ltp/patches/ptrace03.patch [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
++		tst_resm(TINFO,"Running test %d",i);
++		/* since Linux 2.6.26, it's allowed to trace init,
++		   so just skip this test case */
++		if (i == 0 && tst_kvercmp(2, 6, 25) > 0) {
++			tst_resm(TINFO,
++				 "skipping this test as this kernel allows to trace init");
++			continue;
++		}
++
++
```
Comment (shaikshavali1) [inline tests/ltp/patches/ptrace03.patch]: Remove additional line

**Review by shaikshavali1 — COMMENTED** [review]
Please remove additional lines in code

File: tests/ltp/patches/ptrace03.patch (hunk 1 above)
Comment (ashokchandran) [reply tests/ltp/patches/ptrace03.patch]: done

**Review by hukoyu — APPROVED** [review]
LGTM

**Pull request merged**